lib/JITServer/JITServer.cpp

```diff
@@ -397,7 +397,7 @@ ServerNewInterpreterThunkBlock(
 
         // REVIEW: OOP JIT should we clear arena at end?
         ArenaAllocator * arena = scriptContext->GetSourceCodeArena();
-        BYTE * localBuffer = AnewArray(arena, BYTE, bufferSize);
+        BYTE * localBuffer = HeapNewArray(BYTE, bufferSize);
```

Comment:
>localBuffer [](start = 15, length = 11)

how big is the buffer? can use stack? otherwise you need to fix the memory leak for following OOM throw